webpack.prod.js

```diff
@@ -0,0 +1,32 @@
+
+const path = require('path');
+
+module.exports = {
+  mode: 'production',
+  devtool: 'source-map',
```

Comment:
you could probably remove the source-map from production.

Reply:
webpack recommended source-map for production https://webpack.js.org/guides/production/#source-mapping